frappe/gettext/translate.py

```diff
@@ -152,6 +154,51 @@ def directory_filter(dirpath: str | os.PathLike[str]) -> bool:
 		print(f"POT file created at {pot_path}")
 
 
+def get_is_gitignored_function_for_app(app: str | None):
```

Comment:
Can we use [this approach](https://stackoverflow.com/a/70166492) instead of re-implementing git's ignore logic?